PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,24 @@
-<!-- Please prefix the subject above with the Jira issue number. Example: -->
-<!-- [CORE-XYZ] Fixes an issue where ... -->
+<!-- /\ Please prefix the subject above with the Jira issue number. Example: -->
+<!-- [PROJECT-NNN] Fixes an issue where ... -->
+<!-- This should connect your PR to the appropriate JIRA ticket -->
 
-## Jira Issue
-<!-- Please update the link below with the Jira issue number. -->
+<!-- Please update the link below with the Jira project and issue number. -->
 
-[CORE-]
+[PROJECT-NNN](https://lambdaschool.atlassian.net/browse/PROJECT-NNN) Ticket Title / Description
```

Comment:
The link should automatically come through when you use `[PROJECT-NNN]` - Jira Bot automatically links to the right thing. Unless you're wanting to get the Ticket title/description in here, the automatic linking is done.

Reply:
I just edited your PR description, Jjira bot put the link in where I added `[OUTCOMES-241]`

Reply:
:mindblown: That's rad. _updated_